src/index.js

```diff
@@ -57,6 +66,34 @@ class MongoSlowQueryChecker {
    * ```
    */
   async get() {
+    const ops = this.useProfiling
+      ? await this.getFromSystemProfile()
+      : await this.getCurrentlyRunningQueries();
+
+    return this.processOps(ops);
+  }
+
+  /**
+   * Get relevant (slow/COLLSCAN) queries from the `system.profile` table, ensuring we don't
+   * repeatedly return the same queries in consecutive runs.
+   */
+  async getFromSystemProfile() {
+    this.profile = this.profile || (await this.db.connect()).collection('system.profile');
+
+    const conditions = [{ millis: { $gte: this.queryThreshold * 1000 } }];
```

Comment:
We should note that the `system.profile` approach only returns queries longer than both `this.queryThreshold`  and the mongod servers slow query threshold.

Reply:
Done.